docs/guide/ecosystem/documentation.md

```diff
@@ -1,91 +1,227 @@
+
+There are lots of plugins available ranging from accessibility to site layout.
+
+Used by Angular, React and many other big players, it is probably the safest option.
+
+Since designers usually don't like to wreck JavaScript, storybook is the best choice for developers to have authority on how components are showcased, without intervention from designers.
+
+#### Pros
+
+- Big and welcoming community
+- Supports every feature no exception
+- Lots of [plugins](#essential-plugins) from visual testing, to integration with sketch symbols
+- Easy adoption by developers (Examples written in JavaScript)
+
+#### Cons
+
+- Bare bones without plugins (no prop detection, no code samples, no styling)
+- Input and output format not meant for long descriptions
+- Templates are plain text so no syntax highlighting
+- Hard learning curve for designers
+
+#### Useful Links
+
+- [Official Guide](https://storybook.js.org/basics/guide-vue/)
+- [Using Storybook with VueJS](https://auth0.com/blog/using-storybook-with-vuejs/)
+- [Visual testing for Storybook for Vue.js](https://docs.percy.io/docs/storybook-for-vue)
+
+#### Essential Plugins
+
+- [@storybook/addon-a11y](https://github.com/storybooks/storybook/tree/next/addons/a11y) checks accessibility properties
+- [@storybook/addon-knobs](https://github.com/storybooks/storybook) add controls to change usage of your component
+- [@storybook/addon-storysource](https://github.com/storybooks/storybook) shows the source code of the template used for the story
+- [storybook-addon-vue-info](https://github.com/pocka/storybook-addon-vue-info) detects props info
+- [storybook-vue-router](https://github.com/gvaldambrini/storybook-router) add routes to your stories
+
+### Vue Styleguidist
+
+[Vue-styleguidist](https://github.com/vue-styleguidist/vue-styleguidist) is a port of [react-styleguidist](https://github.com/styleguidist/react-styleguidist) for vue components.
+If you are building a Design System and want to showcase the powers of your components, this would probably be your first choice.
 
-### Vue Styledguidist
-https://github.com/vue-styleguidist/vue-styleguidist
+Example: [vue-styleguide](http://rafaelescala.com/vue-styleguide/) or [vue-design-system](https://vueds.com/example/)
+
+It supports all the following vue features:
+
+- Single file components
+- Class style components
+- Functional component templates
+- Template and script commenting (Events, Methods, Props, Slots)
+- JSX & TSX
+- TypeScript & Flow
+- Vuex & Vue-i18n
+- externalized sources for script and template
+- styled components
+
+#### Documentation style
+
+styleguidist uses JSDoc:
+
+```javascript
+props: {
+    /**
+     * The color for the button example
+     */
+    colorExtends: {
+      type: String,
+      default: '#333'
+    }
+  }
+```
+
+#### Pros
+
+- Displays a live editor to try components
+- Meant for design system creation (powers vue-design-system)
+- A strong React community
```

Comment:
Even though its nice to have some community, how would they help if you have a Vue issue?